### PR TITLE
SlackBridge

### PR DESCRIPTION
**Pull request by marceloschmidt**
@RocketChat/core 

For usage instructions, check out https://rocket.chat/docs/administrator-guides/import/slack/slackbridge-md/

### CODE REVIEW

File: packages/rocketchat-slackbridge/slackbridge.js [hunk 1]
```diff
@@ -0,0 +1,711 @@
...
+				msgObj.ets = new Date(parseInt(message.edited.ts.split('.')[0]) * 1000);
+			}
+			if (message.subtype === 'bot_message') {
+				user = RocketChat.models.Users.findOneById('rocket.cat', { fields: { username: 1 } });
+			}
+			RocketChat.sendMessage(user, msgObj, room, { upsert: true });
+		}
+	}
+
+	processSubtypedMessage(room, user, message) {
```
Comment (rodrigok) [inline packages/rocketchat-slackbridge/slackbridge.js]: Codacy Issue found: [This function's cyclomatic complexity is too high. (30)](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2677570450/issues/source?bid=3199634&fileBranchId=3334129#l232)

File: packages/rocketchat-slackbridge/slackbridge.js [hunk 2]
```diff
@@ -0,0 +1,711 @@
...
+			message = '';
+		}
+		return message;
+	}
+
+	findChannel(channelId) {
+		return RocketChat.models.Rooms.findOneByImportId(channelId);
+	}
+
+	addChannel(channelId, hasRetried = false) {
```
Comment (rodrigok) [inline packages/rocketchat-slackbridge/slackbridge.js]: Codacy Issue found: [This function's cyclomatic complexity is too high. (18)](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2677570450/issues/source?bid=3199634&fileBranchId=3334129#l81)

File: packages/rocketchat-slackbridge/slackbridge.js [hunk 3]
```diff
@@ -0,0 +1,711 @@
...
+					return this.uploadFile(details, message.file.url_private_download, user, room, new Date(parseInt(message.ts.split('.')[0]) * 1000));
+				}
+				break;
+			case 'file_comment':
+				logger.events.error('File comment not implemented');
+				return;
+			case 'file_mention':
+				logger.events.error('File mentioned not implemented');
+				return;
+			case 'pinned_item':
```
Comment (rodrigok) [inline packages/rocketchat-slackbridge/slackbridge.js]: Codacy Issue found: [Expected a "break" statement before "case".](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2677570450/issues/source?bid=3199634&fileBranchId=3334129#l311)

File: packages/rocketchat-slackbridge/slackbridge.js [hunk 4]
```diff
@@ -0,0 +1,712 @@
...
+				msgObj.ets = new Date(parseInt(message.edited.ts.split('.')[0]) * 1000);
+			}
+			if (message.subtype === 'bot_message') {
+				user = RocketChat.models.Users.findOneById('rocket.cat', { fields: { username: 1 } });
+			}
+			RocketChat.sendMessage(user, msgObj, room, { upsert: true });
+		}
+	}
+
+	processSubtypedMessage(room, user, message) {
```
Comment (rodrigok) [inline packages/rocketchat-slackbridge/slackbridge.js]: Codacy Issue found: [This function's cyclomatic complexity is too high. (30)](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2677595092/issues/source?bid=3199634&fileBranchId=3334129#l232)

File: packages/rocketchat-slackbridge/slackbridge.js [hunk 5]
```diff
@@ -0,0 +1,712 @@
...
+			message = '';
+		}
+		return message;
+	}
+
+	findChannel(channelId) {
+		return RocketChat.models.Rooms.findOneByImportId(channelId);
+	}
+
+	addChannel(channelId, hasRetried = false) {
```
Comment (rodrigok) [inline packages/rocketchat-slackbridge/slackbridge.js]: Codacy Issue found: [This function's cyclomatic complexity is too high. (18)](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2677595092/issues/source?bid=3199634&fileBranchId=3334129#l81)

**Comment by jmls** [comment]
https://rocket.chat/docs/administrator-guides/import/slack/slackbridge-md/ gives a 404 ...

**Comment by engelgabriel** [comment]
Fixed :)

**Pull request merged**